universum/modules/launcher.py

```diff
@@ -202,10 +203,11 @@ def prepare_command(self) -> bool:  # FIXME: refactor
         return True
 
     def start(self, is_background: bool) -> None:
+        self._is_background = is_background
```

Comment:
Can we move _is_background initialization to the constructor?

Reply:
We only get the real value from `.start()` function parameters, so if I got the question right — no, I don't think we can

Reply:
Read the usage. The distance between constructor call and `.start` call is 3 lines.